parser/src/parser/file.rs

```diff
@@ -327,11 +327,27 @@ impl ParserContext {
             return Ok(members);
         }
 
+        let peeked = &self.peek()?.token;
+        let mut last_constant = matches!(peeked, &Token::Ident(_)) || self.peek_is_function()?;
         let mut last_variable = self.peek_is_function()?;
         let (mut semi_colons, mut commas) = (false, false);
         while self.eat(Token::RightCurly).is_none() {
-            if !last_variable {
-                members.push(self.parse_member_variable_declaration()?);
+            if !last_constant {
+                let (const_, last_const, last_var) = self.parse_const_member_variable_declaration()?;
+                members.push(const_);
+
+                // If it is the last constant update the varaible so we no longer expect consts.
```

Comment:
```suggestion
                // Last constant? Don't expect `const`s anymore.
```

Reply:
The question mark here makes it feel unsure of what the code is doing rather than explaining, in my opinion.

Reply:
Alright